frontend/src/components/c-resizer.vue

```diff
@@ -21,24 +21,25 @@
     slot(name="right")
 </template>
 
-<script>
+<script lang='ts'>
 import { mapState } from 'vuex';
+import { defineComponent } from 'vue';
 
 const DRAG_BAR_WIDTH = 13.25;
 const SCROLL_BAR_WIDTH = 17;
 const GUIDE_BAR_WIDTH = 2;
 
-const throttledEvent = (delay, handler) => {
+const throttledEvent = (delay: number, handler: Function) => {
```

Comment:
```suggestion
const throttledEvent = (delay: number, handler: (event: MouseEvent) => unknown) => {
```
Do you think it will be beneficial to be more strict on the type of the handler, by specifying that it takes in a `MouseEvent` as a parameter?

Reply:
Definitely. Was not aware that of such a possibility while working on the PR. Thanks for pointing this out!

Reply:
ESLint seems to be treating this as an error because the variable `event` in the type definition was not used. I think this case is a clear false positive so it should be fine to just declare an exception to this line (`eslint disable next line`)